applications/serial_lte_modem/doc/SOCKET_AT_commands.rst

```diff
@@ -505,9 +505,34 @@ Syntax
   * ``0`` - Get
   * ``1`` - Set
 
-For a complete list of the supported SET ``<name>`` accepted parameters, see the `SETSOCKETOPT Service Spec Reference`_.
+* The ``<name>`` parameter can accept one of the following values:
 
-For a complete list of the supported GET ``<name>`` accepted parameters, see the `GETSOCKETOPT Service Spec Reference`_.
+  * ``2`` - :c:macro:`SO_REUSEADDR` (set-only).
```

Comment:
Should be some sort of explanation in addition to the macro name if we're adding descriptions to this page.

Reply:
While it prevents duplication and burden on us, it is a lot less usable for the end user as they would basically have to jump between two pages.

Reply:
I can add short descriptions to the options. However, does @lemrey have any input on this? I think I removed the descriptions at some point based on your input.

Reply:
There is no need to duplicate this information, and we shouldn't duplicate it. With you PR, all socket options supported by Modem library are now supported (and the correct level) so you could simply refer to the Modem library documentation for a list of supported socket options.

Reply:
SLM does not support all the options that the modem does and it currently does not have concept of levels in the AT-interface. That is why I wanted to leave the list of supported options also to SLM documentation. Similarly, some (not many) of the options have special functionality in SLM level. See Tomi's [comment](https://github.com/nrfconnect/sdk-nrf/pull/11705/files#r1310323612). 

I also think that relying solely in the modem library documentation is problematic for some parameters, as not all of them are int or char*, but have multitude of structs, so we have to offer an interface between AT-commands and those, so the values are not 1:1 with the modem.

Reply:
I suggest that we leave the list of options and add minimal descriptions to them, or bit more if there is something SLM specific. More information can be seen in the referred nrf_modem documentation.